mantle/coreir/memory.py

```diff
@@ -50,7 +51,7 @@ def CoreirMem(cirb, depth, width):
     return CircuitInstanceFromGeneratorWrapper(cirb, "coreir", "mem", "CoreIRmem_w{}_d{}".format(width, depth),
                                                ["global"], {"width": width, "depth": depth})
 
-def DefineROM(height, width):
+def DefineROM(height, width,read_latency):
```

Comment:
I think we can give read_latency a default value (this caused an existing test to break).

Reply:
Just added default to both DefineROM and DefineRAM